packages/settings-library/src/settings_library/redis.py

```diff
@@ -47,23 +40,3 @@ def _build_redis_dsn(self, db_index: int):
             port=f"{self.REDIS_PORT}",
             path=f"/{db_index}",
         )
```

Comment:
minor: wondering why you removed these helper functions. I find them more self-explanatory. but whatever.